packages/e2e-tests/scripts/buildkite_run_fe_tests.ts

```diff
@@ -208,7 +208,12 @@ export default function run_fe_tests(CHROME_BINARY_PATH, runInCI = true, nWorker
         `${envWrapper} ${path.join(
           "scripts/save-examples.ts"
         )} --runtime=chromium --target=browser ${examplesCfg}`,
-        { cwd: TestRootPath, stdio: "inherit", env: process.env }
+        {
+          cwd: TestRootPath, stdio: "inherit", env: {
+            ...process.env,
+            PLAYWRIGHT_TEST_BASE_URL: "http://localhost:8080"
```

Comment:
NIT: Remove the TODO comment from the other place ([src](https://github.com/replayio/devtools/pull/10287/files#diff-d87fb2b2860caf5159443377e9029f1252c5373f6a5c009aa05871c2f1d4be64R155))?